src/database/schema/ChatSchema.ts

```diff
@@ -284,7 +296,7 @@ class ChatSchema implements BaseDbSchema {
       updatedAt: new Date().toISOString(),
       id: newRandomId,
       type,
-      rawJson: null,
+      rawJson,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is related to the addition of the `json` parameter in the function signature. It's a good practice to store the raw JSON data as it might be useful for debugging or other purposes.
```diff
-      rawJson: null,
+      rawJson,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->